app/models/user.rb

```diff
@@ -1,7 +1,10 @@
 class User < ApplicationRecord
-  has_many :posts, foreign_key: 'author_id'
-  has_many :comments, foreign_key: 'author_id'
-  has_many :likes, foreign_key: 'author_id'
+  has_many :posts, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
+
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
 
   def three_recent_post
```

Comment:
- [ ] As demanded by the project requirements, kindly add unit test for the `three_recent_post` method. You are expected to add unit tests for **all of your models' methods and validations.**